src/main/java/uk/gov/ons/ssdc/exceptionmanager/endpoint/AdminEndpoint.java

```diff
@@ -196,37 +186,6 @@ public void deleteQuarantineRules(@PathVariable("id") String id) {
     cachingDataStore.deleteQuarantineRule(id);
   }
 
-  @Transactional
-  @GetMapping(path = "/replayquarantinedmessage/{id}")
-  public void replaySkippedMessage(@PathVariable("id") String id) {
```

Comment:
We should get a card on the board to implement this endpoint for pubsub messages.
Should be an easier proposition now we own all the topics?